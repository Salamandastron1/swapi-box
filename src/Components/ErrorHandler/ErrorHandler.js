import React, {Component} from 'react';
import App from '../App/App';

class ErrorHandler extends Component {
  constructor() {
    super()
  }
}