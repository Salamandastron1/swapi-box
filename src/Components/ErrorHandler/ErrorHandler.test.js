import React from 'react';
import { shallow, mount, render } from 'enzyme';
import ErrorHandler from './ErrorHandler'

describe('ErrorHandler', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ErrorHandler />);
  });
  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});