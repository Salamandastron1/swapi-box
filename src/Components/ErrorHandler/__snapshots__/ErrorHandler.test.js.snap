// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorHandler matches the snapshot 1`] = `""`;
